tests/testthat/test-automateDataPreparation.R

```diff
@@ -32,7 +32,16 @@ test_that("load sav", {
                                        recodeData = FALSE, recodeMnr = FALSE,
                                        aggregateData = FALSE, scoreData = FALSE,
                                        writeSpss = FALSE, verbose = TRUE),
-               "Please specify oldIDs. Case ID in inputList$savFiles$case.id seems to be empty.", fixed=TRUE)
+               "Please use argument 'oldIDs'. Can't take it from inputList$savFiles$case.id, because at least one case-ID seems to be empty.", fixed=TRUE)
+inputList2 <- inputList
+inputList2$savFiles <- inputList2$savFiles[-c(1:3),] #inputList2$savFiles$case.id is now 'character(0)'
```

Comment:
Tiny nit: Hmmm... Is it really now `character(0)` now 🤔 Wouldn't it be `NULL` as `savFiles` (and therefore `savFiles$case.id`) it does not exist?

If you like to add (not necessary as it works for now), here are some more test cases, adapted to the current version (might be redundant in some way, but at least the case with `case[1] <- NA` would be a new one):

```r
test_that("call automateDataPreparation with missing case.id column or (incomplete) oldIDs", {
  runIncompleteId <- function(inputList) {
    automateDataPreparation(inputList = inputList,
                            datList = inputDat,	path = getwd(),
                            readSpss = FALSE, checkData = TRUE,	mergeData = FALSE,
                            recodeData = FALSE, recodeMnr = FALSE,
                            aggregateData = FALSE, scoreData = FALSE,
                            writeSpss = FALSE, verbose = TRUE)
  }
  
  # savFiles completly missing
  testInputList <- within(inputList, {
    savFiles <- NULL
  })
  expect_error(runIncompleteId(testInputList),
               "Please use argument 'oldIDs'. Can't take it from inputList$savFiles$case.id, because at least one case-ID seems to be empty.",
               fixed = TRUE)
  
  # Only case.id missing
  testInputList <- within(inputList, {
    savFiles$case.id <- NULL
  })
  expect_error(runIncompleteId(testInputList),
               "Please use argument 'oldIDs'. Can't take it from inputList$savFiles$case.id, because at least one case-ID seems to be empty.",
               fixed = TRUE)
  
  # Only some case.id missing
  testInputList <- within(inputList, {
    savFiles$case.id[1] <- NA
  })
  expect_error(runIncompleteId(testInputList),
               "Please use argument 'oldIDs'. Can't take it from inputList$savFiles$case.id, because at least one case-ID seems to be empty.",
               fixed = TRUE)
  
  # Furthermore, value in newID missing
  testInputList <- within(inputList, {
    savFiles$case.id <- NULL
    savFiles$newID$value <- NULL
  })
  expect_error(runIncompleteId(testInputList),
               "Please use argument 'oldIDs'. Can't take it from inputList$savFiles$case.id, because at least one case-ID seems to be empty.",
               fixed = TRUE)
})
```